lib/xcprofiler/reporters/standard_output_reporter.rb

```diff
@@ -13,9 +13,17 @@ def table_for(executions)
         t << ['File', 'Line', 'Method name', 'Time(ms)']
         t << :separator
         executions.each do |execution|
-          t << [execution.filename, execution.line, execution.method_name, execution.time]
+          t << [execution.filename, execution.line, truncate(execution.method_name), execution.time]
         end
       end
     end
+
+    def truncate(text)
+      return text unless text.length >= truncate_at
```

Comment:
I think it is strange depending other property.
`trancate_at` should be passed as the argument.